clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -60,6 +61,7 @@ class AutoScalingWorker {
   private Boolean associatePublicIpAddress
   private String subnetType
   private Integer cooldown
+  private Collection<String> groupMetrics
```

Comment:
I don't see this used anywhere (other than a check as to whether it is groovy-true) - is there a reason this is a Collection<String> vs a Boolean (should the list of groupMetrics be set on the AutoScalingGroup or passed to enableMetricsCollection)?